### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by CoderDS**
Explaining how to avoid a ssh problem on windows.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,6 +10,7 @@
 1. Clone the Octopeer repository to the preferred location and change directory.
 Afterwards, let Vagrant set up a fully running development box and SSH into it.
 (Do this on the host machine, this setup will create a virtual machine itself.)
+When you're using windows, try installing git and running the following commands from Git Bash as 'vagrant ssh' may not work in cmd.
```
Comment (aaronang) [inline README.md]: Use backticks to indicate terminal commands.

windows :arrow_right: Windows
git :arrow_right: Git
and running :arrow_right: , and run the following ...

**Comment by CoderDS** [comment]
Done as you wished.

File: README.md [hunk 2]
```diff
@@ -10,6 +10,11 @@
 1. Clone the Octopeer repository to the preferred location and change directory.
 Afterwards, let Vagrant set up a fully running development box and SSH into it.
 (Do this on the host machine, this setup will create a virtual machine itself.)
+When you're using windows, try installing git and running the following commands from Git Bash as 
+```bash
```
Comment (aaronang) [inline README.md]: I would actually use single backticks in this case.

**Comment by CoderDS** [comment]
Updated.

**Pull request merged**